sormas-api/src/main/java/de/symeda/sormas/api/symptoms/SymptomsDto.java

```diff
@@ -163,6 +163,25 @@
 	public static final String UNEXPLAINED_BLEEDING = "unexplainedBleeding";
 	public static final String UNILATERAL_CATARACTS = "unilateralCataracts";
 	public static final String VOMITING = "vomiting";	
+	public static final String HYDROPHOBIA = "hydrophobia";	
+	public static final String OPISTHOTONUS = "opisthotonus";
+	public static final String ANXIETY_STATES = "anxietyStates";
+	public static final String DELIRIUM = "delirium";
+	public static final String UPROARIOUSNESS = "uproariousness";
+	public static final String PARASTHESIA_AROUND_WOUND = "paresthesiaAroundWound";
+	public static final String EXCESS_SALIVATION = "excessSalivation";
+	public static final String INSOMNIA = "insomnia";
+	public static final String PARALYSIS = "paralysis";
+	public static final String EXCITATION = "excitation";
+	public static final String DYSPHAGIA = "dysphagia";
+	public static final String AEROPHOBIA = "aerophobia";
+	public static final String HYPERACTIVITY = "hyperactivity";
+	public static final String PARESIS = "paresis";
+	public static final String AGITATION = "agitation";
+	public static final String ASCENDING_FLACCID_PARALYSIS = "ascendingFlaccidParalysis";
+	public static final String ERRATIC_BEHAVIOUR = "erraticBehaviour";
+	public static final String COMA = "coma";
+	public static final String DEATH = "death";
```

Comment:
@hzi-braunschweig @bernardsilenou Does "death" really make sense as a symptom?

Reply:
![image](https://user-images.githubusercontent.com/1176692/68320551-12589880-00b8-11ea-89c2-a5611fc760de.png)

Death was mentioned as a symptom (mentioned at three places). Unless I didn't understand what they meant.

Reply:
OK. @bernardsilenou has explained to me. I'm dropping death